openloops.spec

```diff
@@ -23,7 +23,8 @@ link_optimisation = -O2
 EOF
 export SCONSFLAGS="-j %{compiling_processes}"
 ./openloops update --processes generator=0
-./openloops libinstall all.coll
+./openloops libinstall lhc.coll
```

Comment:
@SiewYan why this line is still present? This is not in the master version, and it is not strictly needed at this point I would say. Please clarify

Reply:
Yes, it is not needed. Sorry.

Reply:
@SiewYan ok, but according to your previous answer it does not harm either, right?

Reply:
yes, it does not though, in the future i will be removed.